specification/search/resource-manager/Microsoft.Search/preview/2020-08-01-preview/search.json

```diff
@@ -367,7 +367,7 @@
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The name of the Azure Cognitive Search service to create or update. Search service names must only contain lowercase letters, digits or dashes, cannot use dash as the first two or last one characters, cannot contain consecutive dashes, and must be between 2 and 60 characters in length. Search service names must be globally unique since they are part of the service URI (https://<name>.search.windows.net). You cannot change the service name after the service is created."
+            "description": "The name of the Azure Cognitive Search service to create or update. search service names must only contain lowercase letters, digits or dashes, cannot use dash as the first two or last one characters, cannot contain consecutive dashes, and must be between 2 and 60 characters in length. search service names must be globally unique since they are part of the service URI (https://<name>.search.windows.net). You cannot change the service name after the service is created."
```

Comment:
@arv100kri I think you want to leave two of these "Search" capitalized, as they are the beginning of sentences.